File: TaskBoard.API/Dtos/CardDto.cs
using TaskBoard.API.Database.Entities;

namespace TaskBoard.API.Dtos
{
    public class CardDto : BaseDto
    {
        public string? Name { get; set; }
        public string? Description { get; set; }
        public DateOnly? DueDate { get; set; }
        public int? Priority { get; set; }
        public int? ListId { get; set; }
    }
}

File: taskboard.client/src/Dtos/CardDto.ts
export class CardDto {
    id!: number;
    name!: string;
    description!: string;
    dueDate!: Date;
    priority!: number;
    listId!: number;
}

export enum Priority{
    Low = 1,
    Medium = 2,
    High = 3
}